From bc8d50cb39b925b0bf551a65e2cdf3420c2f77cf Mon Sep 17 00:00:00 2001
From: Stivvo <stivvo01@gmail.com>
Date: Tue, 15 Sep 2020 13:20:05 +0200
Subject: [PATCH] focusmon() moves the cursor accordingly

Moves the cursor to the recently focused monito

Keeps same x and y position relative to the monitor
---
 dwl.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/dwl.c b/dwl.c
index 730e46a1f..f6cdcbb26 100644
--- a/dwl.c
+++ b/dwl.c
@@ -734,9 +734,11 @@ void
 focusmon(const Arg *arg)
 {
 	Client *sel = selclient();
+	Monitor *prevm = selmon;
 
 	selmon = dirtomon(arg->i);
 	focusclient(sel, focustop(selmon), 1);
+	wlr_cursor_move(cursor, NULL, selmon->m.x - prevm->m.x , 0);
 }
 
 void
